.gitignore

```diff
@@ -10,6 +10,14 @@
 # Mobile Tools for Java (J2ME)
 .mtj.tmp/
 
+# IntelliJ_IDEA
```

Comment:
You dont't need ignore both .idea and **/.idea and other simillar ignore. Please leave only relevalent